### ISSUE TITLE
dart:html appears in our API docs in a way that confuses matters

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#72097]
dart:html appears in our API docs in a way that confuses matters. For example, searching for the "Text" widget brings you here:
https://master-api.flutter.dev/flutter/dart-html/Text-class.html

I'm not sure what the solution is; arguably, dart:html is an important library for us to include in our API docs. Maybe it should just be ordered differently? Or in a different color or something?

cc @yjbanov @jcollins-g @gspencergoog
See also https://github.com/dart-lang/dartdoc/issues/2457

**Comment by gspencergoog** [comment]
Yes, it would be good to differentiate it: a different background color for external packages in our search results and ordering them differently seems appropriate.
